README.md

```diff
@@ -137,16 +137,16 @@ attacker. For this reason we advise you not to store the results of
  - [Matthieu Aussaguel](https://github.com/matthieua)
  - [_et al._](https://github.com/envato/zxcvbn-ruby/graphs/contributors)
 
-## License [![license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://github.com/envato/zxcvbn-ruby/blob/master/LICENSE.txt)
+## License [![license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://github.com/envato/zxcvbn-ruby/blob/HEAD/LICENSE.txt)
 
 `zxcvbn-ruby` uses MIT license, the same as [zxcvbn.js][zxcvbn.js] itself. See
-[`LICENSE.txt`](https://github.com/envato/zxcvbn-ruby/blob/master/LICENSE.txt)
+[`LICENSE.txt`](https://github.com/envato/zxcvbn-ruby/blob/HEAD/LICENSE.txt)
```

Comment:
@stevehodgkiss: I'd like to rename the default branch to `main` but I don't seem to have permissions to do so on this repository. Is this something you can do, please?

Reply:
@johnsyweb sorry I can't help - I don't have admin on the repo